addons/crm_iap_lead/models/crm_iap_lead_mining_request.py

```diff
@@ -28,19 +29,31 @@ def _default_lead_type(self):
         else:
             return 'opportunity'
 
+    def _default_country_ids(self):
+        return self.env.user.company_id.country_id
+
+    def _default_user(self):
+        return self.env.user
+
+    @api.depends('user_id')
+    def _compute_team_id(self):
```

Comment:
Guidelines: this should be located below fields definition (globally, we have: default, fields, compute, CRUD, actions).